rocAL/source/meta_data/coco_meta_data_reader.cpp

```diff
@@ -137,8 +148,167 @@ void COCOMetaDataReader::print_map_contents() {
     }
 }
 
+void COCOMetaDataReader::generate_pixelwise_mask(std::string filename, RLE *rle_in) {
+    BoundingBoxCords bb_coords;
+    Labels bb_labels;
+    std::map<int, std::vector<RLE>> FromPoly;
+    auto it = _map_content.find(filename);
+    bb_coords = it->second->get_bb_cords();
+    bb_labels = it->second->get_labels();
+    ImgSize img_size = it->second->get_img_size();
+    MaskCords mask_cords = it->second->get_mask_cords();
+    std::vector<int> polygon_size = it->second->get_polygon_count();
+    std::vector<std::vector<int>> vertices_count = it->second->get_vertices_count();
+    auto &pixelwise_labels = it->second->get_pixelwise_label();
+    int h = img_size.h;
+    int w = img_size.w;
+    pixelwise_labels.resize(h * w);
+    if (rle_in) {
+        for (unsigned int i = 0; i < bb_coords.size(); i++) {
+            bb_labels[i] = _label_info.find(bb_labels[i])->second;
+        }
+    }
+    // Generate FromPoly for all polygons in image
+    int count = 0;
+    for (unsigned int i = 0; i < bb_coords.size(); i++) {
+        for (int j = 0; j < polygon_size[i]; j++) {
+            std::vector<double> in;
+            for (int k = 0; k < vertices_count[i][j]; k++, count++) {
+                in.push_back(mask_cords[count]);
```

Comment:
what is this in variable here?
Please check and change to better variable name if possible